docs/ml-features.md

```diff
@@ -151,7 +151,7 @@ for more details on the API.
  term frequency across the corpus. An optional parameter `minDF` also affects the fitting process
  by specifying the minimum number (or fraction if < 1.0) of documents a term must appear in to be
  included in the vocabulary. Another optional binary toggle parameter controls the output vector.
- If set to true all nonzero counts are set to 1. This is especially useful for modelling discrete
+ If set to true all nonzero counts are set to 1. This is especially useful for discrete
```

Comment:
Let's make this consistent with the doc for `HashingTF` above. 

I'd prefer both to read:

"... optional parameter `binary` controls the output term frequencies. When set to true, all nonzero term frequencies are set to 1. This is especially useful for discrete probabilistic models that model binary, rather than integer, counts.`